UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -207,16 +209,17 @@ public void RestartRound()
 	{
 		if (CustomNetworkManager.Instance._isServer)
 		{
-			if (MapRotationCount <= 10)
-			{
-				CustomNetworkManager.Instance.ServerChangeScene ("Assets/scenes/OutpostDeathmatch.unity");
+			if (MapRotationCount < 10) {
+				CustomNetworkManager.Instance.ServerChangeScene (Scene[1]);
```

Comment:
your variable name is `scenes`, so you should writes `scenes[0]`.

(0 because, in programming you start counting at 0 (= first index in an array))